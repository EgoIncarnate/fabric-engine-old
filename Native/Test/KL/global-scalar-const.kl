const Scalar PI = 3.141;

operator entry()
{
  report PI;
}
