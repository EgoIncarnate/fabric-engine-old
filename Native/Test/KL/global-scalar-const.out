GlobalList
  GlobalConstDecl
    ConstScalarDecl( 'PI', Scalar, 3.141 )
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          Var('PI')
3.141
